meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - python
     - scikit-learn >=0.20
     - scipy >=1.1.0
-    - tensorflow 1.14.*
+    - tensorflow >=2.1.0
     - ipython
     - ipykernel
   run:
```

Comment:
As a question not directly related to this PR, do we really need to duplicate so much between the `host` and `run` sections?

Reply:
Conda's documentation and examples aren't as easy to follow as I'd like - I was thinking the `run_exports` option might help, but I think it could cause other problems.